pygeoapi/api.py

```diff
@@ -1540,9 +1540,8 @@ def get_collection_items(
             if p.uri_field is not None:
                 content['uri_field'] = p.uri_field
             if p.title_field is not None:
-                content['title_field'] = p.title_field
-                content['id_field'] = p.title_field
```

Comment:
Haven't tested it, but are you sure you can delete `content['id_field']` just like that?

Reply:
@tomkralidis good comment by @geosander, i found setting the id_field awkward, because 5 lines before we already set it, it could make sense to set it, if it were still empty. I tried to git blame to find the history of the the line, but it didn't give much clarity, do you remember/know?

Reply:
@pvgenuchten yes, that's the logic (to override).  We use `id-field` in the template so as not to duplicate the visual in the HTML and its associated table.

Reply:
sorry, i don't understand, why is the id-field overriden with the title-field if it exists? where does id-field duplicate the visual?

Reply:
Sorry I was thinking of a downstream application templating use case.

We override the id-field in the items template to display human readable popups when click on a feature.

Reply:
So your suggestion is to restore the override for these cases, let me update the PR